packages/unpublished-test-infra/package.json

```diff
@@ -26,6 +26,7 @@
     "testem": "^3.0.3"
   },
   "devDependencies": {
+    "broccoli-merge-trees": "^4.2.0",
```

Comment:
This needs to be a dependency not a devDep

Reply:
@rwjblue it’s not published

Reply:
trolol, its literally in the file name 🤦